### PR TITLE
Update README + Add Github Templates

### PR DESCRIPTION
**Pull request by AjinkyaIndulkar**
This PR updates the repo README and adds GitHub templates.

#### Added:

- 👷 added GitHub templates for creating issues and pull requests
- 📝 added CHANGELOG

#### Updated:

- 📝 updated README

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ask_a_question.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Ask a Question
+about: Ask a question about using dot
+title: ''
+labels: question
+assignees: ''
```
Comment (giorgiop) [inline .github/ISSUE_TEMPLATE/ask_a_question.md:6]: why including `assignees` to these?

File: .github/ISSUE_TEMPLATE/ask_a_question.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: Ask a Question
+about: Ask a question about using dot
+title: ''
```
Comment (giorgiop) [inline .github/ISSUE_TEMPLATE/ask_a_question.md:4]: we can also remove the `title` IMO

File: .github/ISSUE_TEMPLATE/ask_a_question.md (hunk 1 above)
Comment (AjinkyaIndulkar) [reply .github/ISSUE_TEMPLATE/ask_a_question.md:6]: I'll remove the empty fields.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## :bug: Bug Report
+
+<!-- Note: Remove sections from the template that are not relevant to the issue. -->
+
+### Description:
+
+#### Actual Behavior:
+
+<!-- A clear and concise description of what the bug is. -->
```
Comment (giorgiop) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
<!-- A clear and concise description of what the bug is, including steps for reproducing it. -->
```

**Review by giorgiop — APPROVED** [review]

**Pull request merged**